Sources/AblyAssetTrackingCore/Configuration/ConnectionConfiguration.swift

```diff
@@ -10,14 +10,40 @@ public enum AuthResult {
     case tokenDetails(TokenDetails)
 }
 
+public class ObjcAuthResultJWT: NSObject {
+    public var value: String
+    
+    public init(value: String) {
+        self.value = value
+    }
+}
```

Comment:
How are those changes and all other auth related changes connected to moving the `.xcworkspace` to examples directory? :thinking: Wasn't this committed by mistake? :thinking:

Reply:
No, it wasn't. This change was required to make `PublisherExampleObjectiveC` work, because in objective-c `ConnectionCofiguration` initialiser wasn't visible. There was also a PR for this https://github.com/ably/ably-asset-tracking-swift/pull/194 but I decided to `cherry pick` those changes to the current branch.
Now I know it wasn't the best decision.